source/particles/particle_handler.cc

```diff
@@ -1533,21 +1604,23 @@ namespace Particles
         recv_data_it = static_cast<const char *>(recv_data_it) + cellid_size;
 
         const typename Triangulation<dim, spacedim>::active_cell_iterator cell =
-          triangulation->create_cell_iterator(id);
+          id.to_cell(*triangulation);
```

Comment:
This function has been deprecated. I guess this is accidental revert of the change?